src/Microsoft.Health.Fhir.Core/Features/Operations/Reindex/GetReindexRequestHandler.cs

```diff
@@ -63,6 +63,10 @@ private async Task<GetReindexResponse> GetSingleReindexJobAsync(string jobId, Ca
                 ReindexJobWrapper reindexJob = await _fhirOperationDataStore.GetReindexJobByIdAsync(jobId, cancellationToken);
                 return new GetReindexResponse(HttpStatusCode.OK, reindexJob);
             }
+            catch (JobNotFoundException)
```

Comment:
Seems like `GetReindexJobByIdAsync` can also throw a `RequestRateExceededException`. Do we need to handle that separately or is the `catch(Exception ex)` enough?